libraries/AC_PrecLand/AC_PrecLand_Backend.h

```diff
@@ -15,7 +15,11 @@ class AC_PrecLand_Backend
     // Constructor
     AC_PrecLand_Backend(const AC_PrecLand& frontend, AC_PrecLand::precland_state& state) :
         _frontend(frontend),
-        _state(state) {}
+        _state(state),
+        _los_meas_body(Vector3f{0,  0,  0}),
+        _los_meas_time_ms(0),
+        _have_los_meas(false),
+        _distance_to_target(0.0f) {}
```

Comment:
These are all zeroed unless we're creating one of these objects on the stack, which we shouldn't be.  Save bytes and thinking by removing the zero-initialisation.

Further, we're moving towards member initialisation in the class declaration rather than in the constructor, if it is required.

Reply:
OK, I'll do that. Thanks